recipes/grequests/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "grequests" %}
+{%set version = "0.3.0" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "0f41c4eee83bab39f5543af49665c08681637a0562a5704a3f7b2e4a996531c9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gevent
```

Comment:
Ok, so at this point I'm officially confused. I've had a run of recipes (and will no doubt have similar runs in the future) where `conda build` will tell me that `setuptools` can't be invoked at build time, so some library (e.g. `requests` and `gevent`) must be explicitly included in build. Yet, on these re-runs, I'm *not* seeing those messages. I'm pleased to remove the requirements, but I really don't get what's happening here.